components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -2404,13 +2452,13 @@ private Group addDomainToUserMembers(Group group, String userStoreDomain) throws
             offset = 1;
         }
 
-        // If the results is less than the offset return an empty user list.
+        // If the results are less than the offset or the limit (count) equals to zero, return an empty user list.
         if (offset > users.length) {
```

Comment:
Does this only check for results are less than the offset? if so the  ` or the limit (count) equals to zero,` mentioned in the comment above is misleading

Reply:
fixed: df77256698c74c829147f73d00e2f63979199c16